### DISCUSSION TITLE
how to make a summary and bookmarks in a report

### DISCUSSION THREAD

**Discussion by slhotellier** [eclipse-birt/birt#1404] [Q&A]
Hello 

I'm trying to do some sort of hypertext summary in a report.
But I can't insert a bookmark, even though I have correctly entered the bookmark section.

I have 2 grids, one which contains the summary and the second contains grouped rows which are the destination of the summary.

In the first table I filled in the hyperlink property
In the second I entered the bookmark property

But in the generated html file, I don't have the bookmarks

Is someone have a documentation on it ?

Best regards

**Comment by speckyspooky** [comment]
Could you provide a small example of your report without external data sources.
Normally if the bookmarks are set correctly they can used on the hyperlink to execute such kind of jump.

**Comment by slhotellier** [comment]
Sorry 

I found my mistake due to pre-filled fields and warning message about double quote missing.

Best regards

**Reply by speckyspooky** [reply]
No problem, your are welcome!
